pkg/apis/lockrelease/register.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package lockrelease
+
+// GroupName is the group name used in this package
+const (
+	GroupName = "filestore.csi.storage.gke.io"
```

Comment:
IIRC the group name needs to be unique, using `filestore.csi.storage.gke.io` might result in conflicts in the future, consider using something like `lock.filestore.csi.storage.gke.io`